bundles/binding/org.openhab.binding.tacmi/src/main/java/org/openhab/binding/tacmi/internal/message/AnalogValue.java

```diff
@@ -23,6 +25,7 @@
 public final class AnalogValue {
     public BigDecimal value;
     public TACmiMeasureType measureType;
+    public final MathContext mc = new MathContext(6, RoundingMode.HALF_UP);
```

Comment:
i would assume rounding to more than 2 digits would imply false accurance, right?